python/docs/_templates/card.ejs

```diff
@@ -0,0 +1,30 @@
+```{=html}
+<div class="sd-container-fluid sd-sphinx-override sd-mb-4 gallery-directive docutils">
```

Comment:
I suspect most of this is unnecessary -- it was based on code that I based on code that used sphinx-design to render cards. We should probably just use bootstrap cards, and then this (and much/all of `design-style.css` can go away, I think). At least probaly worth a comment here and/or in design-style so we know to clean that up.